unity/Assets/Scripts/Interfaces/IGameScore.cs

```diff
@@ -0,0 +1,16 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Assets.Scripts.Interfaces
+{
+    public interface IGameScore
+    {
+        void IncreaseScore(int amount);
+        int GetCurrentScore();
+        int GetNextScoreThreshold();
```

Comment:
I assume the Threshold is preparation for adding new tiles to the stack?

Reply:
Yes, this is in preparation for the score based intervals which will allow you to gain additional tiles for the stack.